corehq/apps/data_interfaces/tests/tests.py

```diff
@@ -0,0 +1,127 @@
+from datetime import datetime
+
+from django.test import TestCase
+from django.test import Client
+
+from couchforms.models import XFormInstance
+from corehq.apps.users.models import WebUser
+from corehq.apps.data_interfaces.utils import archive_forms
+from dimagi.utils.excel import WorkbookJSONReader
+
+BASE_PATH = 'corehq/apps/data_interfaces/tests/files/'
```

Comment:
Will this relative path always work (it assumes the cwd)? I prefer building the path off of the current module:
```python
from os.path import abspath, dirname, join
THISDIR = dirname(abspath(__file__))
BASE_PATH = join(THISDIR, 'files')
```